src/js/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,23 @@
+const useLocalStorage = (key) => {
+  const getData = () => {
+    try {
+      const json = localStorage.getItem(key);
```

Comment:
요렇게 한방에 합치면 어떨까요 :) 
사용하지 않는 변수는 선언하기 보다는 바로 inline으로 해줘도 좋을 것 같아요!
```suggestion
      return JSON.parse(localStorage.getItem(key));
```

Reply:
오호 이렇게 써도 되는군요! 참고하겠습니다!😀